Backend/lib/yudhishthira/src/Lib/Yudhishthira/Flow/Dashboard.hs

```diff
@@ -74,16 +74,21 @@ postQueryCreate ::
   Lib.Yudhishthira.Types.ChakraQueriesAPIEntity ->
   m Kernel.Types.APISuccess.APISuccess
 postQueryCreate queryRequest = do
-  existingQueryFields <- getChakraQueryFields queryRequest.chakra
```

Comment:
no need to check duplication, because we use nested object `queryName.fieldName`